src/integrations/feed/clients.ts

```diff
@@ -16,9 +17,20 @@ type FeedFetchOptions = RequestInit & {
 /**
  * Naive implementation of a feed client that fetches the feed from the feed service
  */
-export class FeedClient implements IFeedClient {
+export class FeedClient implements IFeedClient, IGarmrClient {
   private readonly url: string;
   private readonly fetchOptions: FeedFetchOptions;
+  private static readonly garmr = new GarmrService({
```

Comment:
Static so it is the same instance across all FeedClient instances, important so that breaker stats are consolidated across all calls.

Reply:
we still one reused breaker instances but since we want different configurations per different FeedClients moved it to constructor param